### PR TITLE
Add vector-valued gradient example to Autodiff Cookbook

### PR DESCRIPTION
**Pull request by gmittal**
@cghawthorne and I both had difficulty immediately finding an easy way to take vector-valued gradients (which `tf.gradients` provides) and felt it would be useful to provide an example. If there is a better place for this to live (or if you don't feel it's necessary in the docs at all) please let me know.

### CODE REVIEW

**Comment by gnecula** [comment]
Thanks for this contribution. I think that the Autodiff cookbook is a better place for this example: https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html

**Comment by gmittal** [comment]
Great. I've moved the example there.
